wsmgr-cwd panics if run without arguments

wsmgr-cwd panics when run without arguments. I was expecting to get some kind of help output.

```
wsmgr-cwd 
2022/06/16 10:04:49 workspace name = "3"
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.cwd()
	/home/haggi/go/pkg/mod/github.com/stapelberg/wsmgr-for-i3@v0.0.0-20220402161256-eeffe8583511/cmd/wsmgr-cwd/wsmgrcwd.go:63 +0x105
main.main()
	/home/haggi/go/pkg/mod/github.com/stapelberg/wsmgr-for-i3@v0.0.0-20220402161256-eeffe8583511/cmd/wsmgr-cwd/wsmgrcwd.go:71 +0x19
```

--- a/cmd/wsmgr-cwd/wsmgrcwd.go
+++ b/cmd/wsmgr-cwd/wsmgrcwd.go
@@ -58,6 +58,11 @@ func cwd() error {
 		log.Print(err)
 	}
 
+	// check if we have something to run
+	if len(os.Args[1:]) < 2 {
+		log.Fatal("no command to execute")
+	}
+
 	// run the remaining command line args
 	args := os.Args[1:]
 	full, err := exec.LookPath(args[0])

--- a/cmd/wsmgr-cwd/wsmgrcwd.go
+++ b/cmd/wsmgr-cwd/wsmgrcwd.go
@@ -59,7 +59,7 @@ func cwd() error {
 	}
 
 	// check if we have something to run
-	if len(os.Args[1:]) < 2 {
+	if len(os.Args) < 2 {
 		log.Fatal("no command to execute")
 	}
 

--- a/cmd/wsmgr-cwd/wsmgrcwd.go
+++ b/cmd/wsmgr-cwd/wsmgrcwd.go
@@ -60,7 +60,7 @@ func cwd() error {
 
 	// check if we have something to run
 	if len(os.Args) < 2 {
-		log.Fatal("no command to execute")
+		log.Fatalf("no command to execute: syntax: %s <command>", os.Args[0])
 	}
 
 	// run the remaining command line args

